src/Microsoft.Framework.ConfigurationModel/Properties/Resources.Designer.cs

```diff
@@ -91,6 +91,22 @@ internal static string FormatError_InvalidFilePath()
         }
 
         /// <summary>
+        /// Unable to resolve path '{0}'; construct this {2} with a non-null {1}.
+        /// </summary>
+        internal static string Error_MissingBasePath
+        {
+            get { return GetString("Error_MissingBasePath"); }
+        }
+
+        /// <summary>
+        /// Unable to resolve path '{0}'; construct this {2} with a non-null {1}.
+        /// </summary>
+        internal static string FormatError_MissingBasePath()
```

Comment:
Manually. My tooling didn't update this file. Is it supposed to be updated automatically?